app/controllers/steps/submission/review_controller.rb

```diff
@@ -17,19 +17,13 @@ def update
 
       def set_presenter
         @presenter = Summary::HtmlPresenter.new(
-          crime_application: JSON.parse(draft_json_application)
+          crime_application: current_crime_application.draft_submission
         )
       rescue Dry::Struct::Error => e
         Rails.error.report(e, handled: true)
 
         redirect_to edit_crime_application_path(current_crime_application)
       end
-
-      def draft_json_application
```

Comment:
Moved to the CrimeApplication model as we're creating draft applications in multiple places now.